src/harness/BenchmarkDotNet.Extensions/RecommendedConfig.cs

```diff
@@ -99,56 +99,52 @@ private static IEnumerable<string> GetBenchmarksToResume(DirectoryInfo artifacts
                 return new string[0];
 
             // Get all existing report files, of any export type; order by descending filename length to avoid rename collisions
-            var existingBenchmarks = artifacts.GetFiles($"*-report-*", SearchOption.AllDirectories)
-                .OrderByDescending(resultFile => resultFile.FullName.Length)
+	    var toRename = artifacts.GetFiles("*-report-*", SearchOption.AllDirectories)
+		    .Where(resultFile => !resultFile.FullName.Contains("-resume-report-") || File.Exists(resultFile.FullName.Replace("-resume-report-", "-report-")))
```

Comment:
Is this removal of the "-resume-report-" part of the name accounting for a user resuming multiple times?

Reply:
Yeah, this is doing:

1. Get all files matching `*-report-*`, which would include `-*resume-report-*`
2. Filter to those that _do not_ already have `-resume-report-` in their name (so they are just `-report-`), **OR** those that _do_ already have `-resume-report-` in the name, and there's also a `-report-` file sitting behind it waiting to get renamed

When there's already a `-report-` and `-resume-report`, then the `-resume-report` file will get renamed to `-resume-resume-report` and `-report-` will get renamed to `-resume-report-` as a result. But if there's a `-resume-report-` and there isn't a `-report-` to match, then we don't need to rename it to `-resume-resume-report-` because there won't be a collision.